### PR TITLE
orphan collection of NIC during Integration Tests

### PR DESCRIPTION
**Pull request by Mkmittal**
**What this PR does / why we need it**:
Adds orphan Collection of NIC during Integration tests using tags.
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```doc developer
NICs are tagged during integration tests
```

### CODE REVIEW

**Comment by gardener-robot-ci-3** [comment]
Thank you @Mkmittal for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: test/integration/provider/rti.go [hunk 1]
```diff
@@ -40,19 +40,19 @@ func (r *ResourcesTrackerImpl) probeResources() ([]string, []string, []string, [
 	// Describe volumes attached to VM instance & delete the volumes
 	// Finally delete the VM instance
 
-	clusterTag := "tag:kubernetes.io/cluster/" + r.ClusterName
-	clusterTagValue := "1"
+	//clusterTag := "tag:kubernetes.io/cluster/" + r.ClusterName
+	//clusterTagValue := "1"
```
Comment (himanshu-kun) [inline test/integration/provider/rti.go:44]: remove it if its not used now

File: test/integration/provider/aws.go [hunk 2]
```diff
@@ -192,51 +192,29 @@ func getOrphanedNICs(tagName string, tagValue string, machineClass *v1alpha1.Mac
 	var orphanNICs []string
 	sess := newSession(machineClass, &v1.Secret{Data: secretData})
 	svc := ec2.New(sess)
-	inputVPC := &ec2.DescribeVpcsInput{
+	inputNI := &ec2.DescribeNetworkInterfacesInput{
```
Comment (himanshu-kun) [inline test/integration/provider/aws.go:195]: inputNI -> inputNIC

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Review by himanshu-kun — CHANGES_REQUESTED** [review]
kindly address the comments otherwise lg

File: test/integration/provider/aws.go (hunk 2 above)
Comment (Mkmittal) [reply test/integration/provider/aws.go:195]: ok
Done

File: test/integration/provider/rti.go (hunk 1 above)
Comment (Mkmittal) [reply test/integration/provider/rti.go:44]: Done

**Comment by Mkmittal** [comment]
@himanshu-kun, the requested changes have been made.

**Review by himanshu-kun — APPROVED** [review]
/lgtm

**Pull request merged**